workspaces/liferay-testray-workspace/client-extensions/liferay-testray-custom-element/src/context/TestrayContext.tsx

```diff
@@ -5,7 +5,7 @@
  * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
  */
 
-import {ReactNode, createContext, useEffect, useMemo, useReducer} from 'react';
+import {createContext, useEffect, useMemo, useReducer} from 'react';
```

Comment:
Should we add `React.ReacNode` in [line 120](https://github.com/liferay-frontend/liferay-portal/pull/4441/files#diff-a388ae1326c3bd61a1c27a2cc260ef3dee5ee514bbdd7c8dd8193df790ef0dfaR120)?